package.json

```diff
@@ -49,7 +50,8 @@
     "gulp-watch": "^5.0.1",
     "jshint": "^2.9.7",
     "pump": "^3.0.0",
-    "sinon": "^4.5.0"
+    "sinon": "^4.5.0",
+    "typescript": "3.0.1"
```

Comment:
Oldest stable version of TypeScript `3.x.x` to ensure that the written `index.d.ts` is compatible with all 3+ versions.